# Nils Freydank <holgersson@posteo.de> (2020-01-08)
Avoid auto installation of files we install later manually.
Adopted from asturm's patch for mdds-1.4.3.

--- a/Makefile.am       2020-02-08 20:00:00.000000000 +0100
+++ b/Makefile.am	    2020-02-08 20:00:00.000000001 +0100
@@ -4,7 +4,6 @@
 
 AM_CPPFLAGS = -I$(top_srcdir)/include $(CXXFLAGS_UNITTESTS)
 
-dist_doc_DATA = AUTHORS README.md
 nodist_pkgconf_DATA = misc/mdds-@API_VERSION@.pc
 
 DISTCLEANFILES = \
@@ -14,8 +13,6 @@
 
 EXTRA_DIST = \
 	autogen.sh \
-	CHANGELOG \
-	LICENSE \
 	doc/conf.py \
 	doc/doxygen.conf \
 	doc/flat_segment_tree.rst \
@@ -164,14 +161,6 @@
 	rtree_test_bulkload_mem.mem
 endif
 
-install-data-local:
-	$(MKDIR_P) $(DESTDIR)$(docdir)
-	$(INSTALL_DATA) $(top_srcdir)/LICENSE $(DESTDIR)$(docdir)/COPYING
-	$(INSTALL_DATA) $(top_srcdir)/CHANGELOG $(DESTDIR)$(docdir)/NEWS
-
-uninstall-local:
-	rm -f $(DESTDIR)$(docdir)/COPYING $(DESTDIR)$(docdir)/NEWS
-
 if BUILD_DOCS
 html-local:
 	(cd doc && $(DOXYGEN) doxygen.conf)

